ihontaryk/homework_11/guess_word.py

```diff
@@ -0,0 +1,129 @@
+import random
+import csv
+from enum import Enum
+from pathlib import Path
+import re
+
+
+class CsvFileReader:
+    """
+    Class for reading and modifying CSV file
+    """
+
+    def __init__(self, csv_file, directory="homework_11"):
```

Comment:
do not hardcode such things here, 
let it be None, and in body you can get current directory